lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -137,16 +144,17 @@
                 }
             ];
 
-            if (_.isArray(authData.providers)) {
-                var accountsSectionData = {
-                    title: gettext('Connected Accounts'),
+            accountsSectionData = [
+                {
+                    title: gettext('Linked Accounts'),
+                    subtitle: gettext(
```

Comment:
@waheedahmed This message has the same problem as my previous comment, i.e. the text should not be split across multiple lines, and edX should not be included in the message.